appveyor.yml

```diff
@@ -27,21 +27,15 @@ init:
 
 install:
   - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-  - cmd: conda update --yes --quiet conda
+  - cmd: conda config --set always_yes yes
+  - cmd: conda install -q conda=4.5
   - cmd: set PYTHONUNBUFFERED=1
   - cmd: conda info
-  - cmd: conda config --set always_yes yes
-  - cmd: conda update conda
+  - cmd: conda config --add channels conda-forge
   - cmd: conda install -q --force --no-deps psutil ruamel_yaml requests
```

Comment:
@mcflugen Do you know if I need this line?

I have it because the Landlab build script had it.

Reply:
We think likely not necessary